### DISCUSSION TITLE
下载的v3.15.0 alist-darwin-amd64.tar.gz解压后无法运行

### DISCUSSION THREAD

**Discussion by 1265578519** [AlistGo/alist#4043] [Q&A]
下载的v3.15.0 alist-darwin-amd64.tar.gz解压后无法运行

什么原因？电脑是amd64位的
![image](https://user-images.githubusercontent.com/6442439/229943483-d8538a8d-e298-47dd-af2a-971a63e2bc0a.png)

**Comment by 1265578519** [comment]
解决了，，电脑上要下载这个alist-desktop_3.15.0_x64_en-US.msi
https://github.com/alist-org/desktop-release/releases/latest/download/alist-desktop_3.15.0_x64_en-US.msi

**Reply by a447948814** [reply]
openwrt下载的.gz文件如何安装呢
